lua/apisix/admin/plugins.lua

```diff
@@ -42,4 +42,22 @@ function _M.check_schema(plugins_conf)
 end
 
 
+function _M.get(name)
```

Comment:
bad name, how about `get_schema` ?

Reply:
no, you can not change the name, which is from the HTTP method.